sw/host/hsmtool/scripts/BUILD

```diff
@@ -0,0 +1,117 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@rules_python//python:defs.bzl", "py_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+# Since rust seems generally unfriendly to generated code, the PKCS#11 constants
+# should be regenerated whenever we change the version of the rust-cryptoki library
+# and then copied into //sw/host/hsmtool/src/util/attribute
+filegroup(
+    name = "generated_srcs",
```

Comment:
What is the purpose of the `generated_srcs` filegroup rule? My understanding is that because it is `manual` it cannot be grouped into any higher-level `filegroup` rules, so it doesn't seem very useful.

I'm probably just missing something about how Bazel works.

Reply:
Oh I see, thanks for the explanation!